snap/snapcraft.yaml

```diff
@@ -13,13 +13,17 @@ confinement: strict
 
 parts:
     novnc:
-        source: build/
+        source: https://github.com/novnc/noVNC/archive/v1.2.0.tar.gz
         plugin: dump
         stage-packages:
-            - websockify
             - bash
             - jq
-            - python-numpy
+            - python3-numpy
+            
+    websockify:
+        source: https://github.com/novnc/websockify/archive/v0.9.0.tar.gz
+        plugin: python
+        stage-packages:
             - python3-numpy
```

Comment:
I assume this means numpy should be removed earlier?

Reply:
Hmm, does novnc use numpy at all? If not then it should be removed, I had it in there previously as it was needed for the Debian/Ubuntu websockify package